nexus/db-queries/src/db/lookup.rs

```diff
@@ -453,7 +483,7 @@ impl<'a> Root<'a> {
 lookup_resource! {
     name = "Silo",
     ancestors = [],
-    children = [ "IdentityProvider", "SamlIdentityProvider", "Project" ],
+    children = [ "IdentityProvider", "SamlIdentityProvider", "Project", "SiloImage" ],
```

Comment:
Should `Image` be added here? cc @davepacheco

Reply:
I believe the only purpose of putting things here is to generate selectors on this lookup type that will look up child resources.  e.g., if you wanted to do `LookupPath(...).silo_id(...).image_name(...)`, then you'd want to put that here.  I haven't looked at the rest of this PR yet to know if you'd want that here but it seems like not since you should already have `SiloImage`.  But I think it's safe to leave it out if you're not sure.

Reply:
This base `Image` type will only ever be used to access an image on ID so it seems fine.